### PR TITLE
Created a PR for DragonCurve TDD

### PR DESCRIPTION
**Pull request by stitsyuk**

### CODE REVIEW

File: Sedgewick/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,63 @@
+###############################################################################
```
Comment (pharrukh) [inline Sedgewick/.gitattributes]: Make several commits, so that what is not related to the main change will not be mixed with important changes.

File: Sedgewick/TDD/DragonCurve.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            var actualCollection = Functions.DragonCurve(4);
+            Assert.AreEqual(expectedCollection, actualCollection);
+        }
+        [TestMethod]
+        public void DragonCurveTest5()
+        {
+            var expectedCollection = "FLFLFRFLFLFRFRFLFLFLFRFRFLFRFRFLFLFLFRFLFLFRFRFRFLFLFRFRFLFRFRF";
+            var actualCollection = Functions.DragonCurve(5);
+            Assert.AreEqual(expectedCollection, actualCollection);
+        }
```
Comment (pharrukh) [inline Sedgewick/TDD/DragonCurve.cs]: Tests are correct. 👍

File: Sedgewick/TDD/DragonCurve.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    }
+
+    public static class Functions
```
Comment (pharrukh) [inline Sedgewick/TDD/DragonCurve.cs]: Where are the arrays? The task was to use arrays in the solution.

File: Sedgewick/.gitattributes (hunk 1 above)
Comment (pharrukh) [reply Sedgewick/.gitattributes]: Это пока ты знаешь, я так понял, раз проигнорировал.

Comment (stitsyuk) [reply Sedgewick/.gitattributes]: :(
